schematic_api/api/openapi/api.yaml

```diff
@@ -9,12 +9,20 @@ info:
 servers:
   - url: /v1
 
+components:
+  securitySchemes:
+    api_key:
+      type: apiKey
```

Comment:
Thanks for the question Tom. I read the documentation that you linked there and tried the approach, but it didn't work (even when I provided a token, the API would still respond with 401 client error). I then noticed that for schematic, we are using connexion (see documentation here: https://connexion.readthedocs.io/en/stable/security.html) which has a different way of handling security than swagger. 

Connexion provides an example of using [jwt](https://github.com/spec-first/connexion/blob/main/examples/jwt/app.py) and [api key](https://github.com/spec-first/connexion/blob/main/examples/apikey/app.py), and it seems like the JWT approach would require generating a secret? I don't think that's what we would need and it also didn't work when I tried implementing it.. I then looked at the approaching of using API key, the implementation requires structuring token to something like `TOKEN_DB = {token: {"uid": 'user id'}}` because I guess they are thinking of having a unique token for each users based on their user id, and I tweaked it a bit to make it work for our use case. 

I am not sure if this is the best practice at all since the implementation does seem a bit strange.

Reply:
@thomasyu888 Very interestingly, when I experimented with Thomas's mono repo with open api generator, the approach of JWT seems to work: https://github.com/Sage-Bionetworks/sage-monorepo/pull/1599/files#diff-4a5248631e7d0aeeb09e3170c8eb12539cfca70dc7e63214532426fa2f8f7d39